src/shared/models/fault-marking.model.ts

```diff
@@ -22,3 +22,11 @@ export enum CommentSource {
 }
 
 export type FaultSummary = Competency & MultiFaultAssignableCompetency & CommentedCompetency;
+
+export enum CompetencyIdentifiers {
+  COMMENTS = 'Comments',
```

Comment:
Sorry I know its pedantic but can we call this `COMMENTS_SUFFIX` and the fault one `FAULT_SUFFIX` just for clarity.

Reply:
done